pkg/transformers/pathconfig.go

```diff
@@ -25,6 +25,8 @@ import (
 type PathConfig struct {
 	// If true, it will create the path if it is not found.
 	CreateIfNotPresent bool
+	// If true, the path will be ignored if the path is found.
+	IgnoreIfPresent bool
```

Comment:
This seems necessary in order to support paths in cluster-scoped resources.  If we didn't make an exception in the `ClusterRoleBinding` case, we would prevent specifying a service account as a subject that is not part of the kustomization namespace (see test case).